### ISSUE TITLE
Workchain phpstorm und github läuft noch nicht richtig

### ISSUE THREAD

**Issue by bergischweb** [T3DocumentationStarter/Public-Info-015#1]
Hallo Martin, 

vielleicht kannst du mir weiterhelfen. Ich meine gesehen zuhaben, dass du auch mit PhpStorm arbeitest. 

Ich habe das Repository in PhpStorm reingezogen, aber ich bekomme es nicht hin, die dort gemachten Änderungen wieder hochzuladen. Ich bekomme zwar die Meldung: "hat alles geklappt", aber wenn ich dann in github nachsehe, hat sich überhaupt nichts getan.

Hast du ein "Kochrezept", wie man das macht? Ich mach da wohl irgend so einen blöden Denkfehler, der nirgendwo beschrieben ist. Ich komme nicht dahinter. 

Beste Grüße
Stefan

**Comment by marble** [comment]
Ja, klar, mach ich so. Guck mal, was Mattes hier geschrieben hat: https://docs.typo3.org/typo3cms/drafts/github/wmdbsystems/RSTingWithPhpStormGuide/ Ich habe keine Zeit, mir das jetzt durchzulesen. Ich schreib's in die ToDo-Liste, vielleicht finden wir dann einen der neuen Doku-Mitstreiter dass er hilft, das in eine "offizielle" Doku umzuwandeln oder aufzunehmen.

Im Prinzip mache ich: System=Linux bei mir, Git einrichten, Schlüssel nach Github hochladen, in PHPStorm das:
- Plugin installieren ![023](https://cloud.githubusercontent.com/assets/307057/20070991/ebe10528-a522-11e6-82f7-9ad3720831d1.png)
- Tab und Indent auf 3 Blanks einstellen 
  ![024](https://cloud.githubusercontent.com/assets/307057/20071065/353b4efe-a523-11e6-8305-c9e9e143417d.png)
- Ggf. Settings/Version Control einstellen. Ich musste da nichts machen.
- Wichtig: Beim jeweiligen Projekt unter Settings alle Repository eintragen 
  ![025](https://cloud.githubusercontent.com/assets/307057/20071232/db2b24e2-a523-11e6-87da-ffcdbe15c500.png)
  Dann hast du im Projektbaum im Kontextmenü einen Menüpunkt "Git"
- Oft öffne ich aber auch ein Terminal-Pane in PHPStorm, "zieh den Ordner darein" - dann stehe ich im richtigen Verzeichnis - und tippe Befehle ein: 
  ![026](https://cloud.githubusercontent.com/assets/307057/20071336/5d9e666e-a524-11e6-811a-42b647eabf7b.png)
- Guck mal ggf. in der PhpStorm-Hilfe bzw. in der Jetbrains Videothek nach.
- Sag später mal, was hilft.

Ich hoffe, du kommst so schon weiter!

**Comment by marble** [comment]
Daniel Lorenz hatte das Problem auch. Und zwar macht er GIT auf der Kommandozeile. Und da hat sich gezeigt, dass er seinen Schlüssel, den er bei Github hinterlegt hat, noch einmal authorisieren musste. Danach klappte es. Er hat auf der Kommandozeile also folgendes gesehen:

![bildschirmfoto_2016-11-08_13-12-07](https://cloud.githubusercontent.com/assets/307057/20103962/4a917c7a-a5cc-11e6-959f-005f669a2080.png)

Dann ist er zur angezeigten URL gegangen und hat den Schlüssel noch einmal akzeptiert.
- **Vielleicht kannst du das bestätigen?**
- **und hier per Screenshot dokumentieren?**

Dann könnten wir das später gleich in die Doku übernehmen. Für jetzt verlinke ich diesen Issue 
https://github.com/T3DocumentationStarter/Public-Info-015/issues/1 erstmal hier https://docs.typo3.org/typo3cms/drafts/github/T3DocumentationStarter/Public-Info-001/AgendaDocumentation/Index.html#how-to-do-documentation-with-github
